src/main/java/com/internetitem/logback/elasticsearch/writer/ElasticsearchWriter.java

```diff
@@ -52,7 +62,27 @@ public void sendData() throws IOException {
 			return;
 		}
 
-		HttpURLConnection urlConnection = (HttpURLConnection)(settings.getUrl().openConnection());
+		HttpURLConnection urlConnection;
+
+		URL url = settings.getUrl();
+
+		if ("https".equals(url.getProtocol().toLowerCase())){
+
+			trustAllHosts();
+
+			HttpsURLConnection httpsURLConnection = (HttpsURLConnection) url.openConnection();
+			httpsURLConnection.setHostnameVerifier(SSLConnectionSocketFactory.ALLOW_ALL_HOSTNAME_VERIFIER);
+
+			if (url.getUserInfo() != null) {
```

Comment:
Maybe store username/password as separate configuration settings, and if set then do this? I'm just imagining that some people may store the URL in one place and the credentials separately